libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -64,6 +68,9 @@
     #error "IOT_MQTT_RETRY_MS_CEILING cannot be 0 or negative."
 #endif
 
+#define NETWORK_BUFFER_SIZE    ( 1024U )
```

Comment:
make it a default value instead a define. It will enable us to set the config by an application if needed.